src/img/fingerprint-icon.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 42 48">
```

Comment:
Maybe a broader discussion than we need to solve here, but I wonder if we should exclude these sorts of images from the design system package, since they're pretty use-case-specific. We have a few already though, so I think it'd be fine to follow the pattern.

Reply:
I think it would be good to exclude because it would help the package size.

Reply:
If we wanted to do it as part of this pull request, we could put the file somewhere in `docs`, like `docs/img/fingerprint-icon.svg`, then reference it in the documentation as `/img/fingerprint-icon.svg`.